### ISSUE TITLE
Asus X206H closes USB Port

### ISSUE THREAD

**Issue by VirToReal** [Grippentech/Asus-E200HA-Linux-Post-Install-Script#23]
Hi!

Used the script to install new Kernel, sound works perfectly fine after a few attempts where PulseAudio wants to boot up. Before that, the second USB Port on my Asus X206H is working. After PulseAudio is able to setup the Server, sound and server is running, but USB Port won't work anymore. 

The time-delay until "sound is working" / "one USB-Port shutdown" is about a few minutes after OS Bootup. (Kubuntu 16.10)

uname -a:

> Linux hardy-eeeh 4.12.0-rc1-custom3 #2 SMP Mon Jun 5 16:58:48 EDT 2017 x86_64 x86_64 x86_64 GNU/Linux

But I think this is a Kernel issue right?

**Comment by Grippy98** [comment]
Yeah it sounds that way. Haven't seen it before. Have you tried looking at dmesg output to check for any reported errors that might be sound or USB related?
